lib/src/typedefs.dart

```diff
@@ -32,4 +32,8 @@ typedef PageChangeCallback = void Function(
   CalendarEventData event,
 );
 
-typedef StringProvider = String Function(DateTime date);
+typedef StringProvider = String Function(DateTime date,
+    [DateTime? secondaryDate]);
```

Comment:
Can we use named parameter for `secondaryDate`?
Ex : `static String _dayStringBuilder(DateTime date, {DateTime? secondaryDate})`